docs/guide.rst

```diff
@@ -1099,6 +1096,30 @@ For example, to treat warnings as errors and run in nit-picky mode:
 This Nox session always runs with the current major release of Python.
 
 
+.. _The docs-build session:
+
+The docs-build session
+----------------------
+
+Build the documentation using the Nox session ``docs-build``:
+
+.. code:: console
+
+   $ nox --session=docs-build
+
+The docs session runs the command ``sphinx-build``
+to generate the HTML documentation from the Sphinx directory.
+
+Use the ``--`` separator to pass additional options to either tool.
```

Comment:
```suggestion
Use the ``--`` separator to pass additional options.
```

Please also apply this change (remove "to either tool") to the section about the `docs` session.

Reply:
**Update:** Actually, please remove this line from the section about `docs-build`. But do include it in the section about `docs`, without the text `"to either tool"`.

(Sorry, been a bit tricky to do this with inline suggestions...)

Reply:
No problem, I don't even know how to do this inline reviews yet. Everything coming out in one minute.